processing/src/main/java/io/druid/segment/virtual/ExpressionColumnValueSelector.java

```diff
@@ -82,6 +82,6 @@ public void inspectRuntimeShape(RuntimeShapeInspector inspector)
   @Override
   public boolean isNull()
   {
-    return getObject().isNull();
+    return getObject().isNumericNull();
```

Comment:
Why this?

Reply:
It is possible for an expression to have a non-null String value but it can return null when parsed as a primitive long/float/double.
   ExprEval.isNumericNull checks whether the parsed primitive value is null or not.

Reply:
added comment to code.